### PR TITLE
add typing skip_gram function and remove typing info file

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Please keep the file, it'll need to be there when people add new features to the API :)

**Comment by gabrieldemarmiesse** [comment]
You can just have an empty list for exceptions.

File: tensorflow_addons/text/skip_gram_ops.py [hunk 1]
```diff
@@ -19,27 +19,30 @@
...
-    seed=None,
-    name=None,
-):
+    input_tensor: TensorLike,
+    min_skips: FloatTensorLike = 1,
+    max_skips: FloatTensorLike = 5,
+    start: FloatTensorLike = 0,
+    limit: FloatTensorLike = -1,
+    emit_self_as_target: bool = False,
+    vocab_freq_table: AcceptableDTypes = None,
```
Comment (gabrieldemarmiesse) [inline tensorflow_addons/text/skip_gram_ops.py]: It seems to be a lookup table, not a data type.

Comment (autoih) [reply tensorflow_addons/text/skip_gram_ops.py]: Thanks @gabrieldemarmiesse. I added `tf.lookup.KeyValueTensorInitializer` in `AcceptableDTypes`.  Maybe just place here directly.

Comment (gabrieldemarmiesse) [reply tensorflow_addons/text/skip_gram_ops.py]: Indeed. Having `vocab_freq_table=tf.float32` wouldn't be correct for exemple.

Comment (autoih) [reply tensorflow_addons/text/skip_gram_ops.py]: Got it. Thanks @gabrieldemarmiesse.

**Comment by gabrieldemarmiesse** [comment]
Thanks a lot!

**Pull request merged**